### PR TITLE
Check blackdoc in CI.

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
## Motivation

Check the docstring by black in CI.

## Description of the changes

Run blackdoc in CI.

### CODE REVIEW

**Review by HideakiImamura — COMMENTED** [review]
Thanks for the PR. The introduced `blackdoc` looks good to me.

Currently, PR #1755 is ongoing and about to be merged. I think this PR should be merged after merging #1755.

**Review by hvy — APPROVED** [review]
LGTM!

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def get_tests_require() -> List[str]:
 def get_extras_require() -> Dict[str, List[str]]:
 
     requirements = {
-        "checking": ["black", "hacking", "isort", "mypy==0.782"],
+        "checking": ["black", "hacking", "isort", "mypy==0.782", "blackdoc"],
```
Comment (toshihikoyanase) [inline setup.py:53]: [Note] The license of `blackdoc` is the [MIT](https://github.com/keewis/blackdoc/blob/master/LICENSE) license. I think we can use it in terms of the license.

**Comment by HideakiImamura** [comment]
@keisuke-umezawa PR #1755 has been merged. Could you update this PR to align the current master? I think we need to update `checks.sh` and `CONTRIBUTING.md` appropriately.

**Review by toshihikoyanase — APPROVED** [review]
LGTM except for the conflict. Thank you for your contribution.

**Comment by keisuke-umezawa** [comment]
@HideakiImamura I updated `checks.sh`. Could you check it?

**Review by HideakiImamura — APPROVED** [review]
Thanks for the swift actions! LGTM!

[Just a note] I found that the current `blackdoc` does not support the `--diff` option while `black` do. I think we should add the `--diff` option to `blackdoc` if it supports the option in the future.

**Pull request merged**